pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "wyvern-ai"
-version = "0.0.28"
+version = "0.0.27"
```

Comment:
shall we do `0.0.29`?

Reply:
i thought it'd be fine since we didn't release it to pypi